examples/benchmark/plot_benchmark_results.py

```diff
@@ -0,0 +1,99 @@
+# Authors: Florian Hofer
+#
+# License: BSD (3-clause)
+
+"""Plot results of runtime and detection quality benchmarks."""
+
+import sys
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+
+results_filepath = Path(sys.argv[1])
+benchmark, db_slug, *_ = results_filepath.stem.split('__')
+plot_filepath = results_filepath.with_suffix('.svg')
+results = pd.read_csv(results_filepath).sort_values('detector')
+
+if benchmark == 'runtime':
+    results['signal_len'] = results['num_samples'] / (results['fs'] * 3600)
+    results['runtime_per_minute'] = results['runtime'] / results['signal_len'] / 60  # noqa
+    results = results.groupby(['detector', 'signal_len'], as_index=False).agg(
+        mean_runtime=('runtime', 'mean'),
+        std_runtime=('runtime', 'std'),
+        mean_runtime_per_minute=('runtime_per_minute', 'mean'),
+        std_runtime_per_minute=('runtime_per_minute', 'std'),
+        n=('runtime', 'count'),
+    )
+    results['error'] = results['std_runtime'] / np.sqrt(results['n'])
+    results['error_per_minute'] = results['std_runtime_per_minute'] / np.sqrt(results['n'])
+
+    fs = results['fs'][0]
+
+    fig = px.line(
+        results,
+        x='signal_len',
+        y='mean_runtime',
+        error_y='error',
+        color='detector',
+        log_y=True,
+        labels={
+            'signal_len': 'signal length in hours',
+            'mean_runtime': 'mean runtime in s',
+        },
+        title=f'Mean detector runtime for {db_slug.upper()} (fs={fs}Hz)',
+        width=800,
+        height=600,
+        render_mode='svg',
+    )
+    fig.update_yaxes(rangemode='tozero')
+    fig.write_image(plot_filepath)
+
+    fig = px.line(
+        results,
+        x='signal_len',
+        y='mean_runtime_per_minute',
+        error_y='error_per_minute',
+        color='detector',
+        log_y=True,
+        labels={
+            'signal_len': 'signal length in hours',
+            'mean_runtime_per_minute': 'mean runtime per minute in s',
+        },
+        title=f'Mean detector runtime per signal minute for {db_slug.upper()} (fs=128Hz)',
+        width=800,
+        height=600,
+        render_mode='svg',
+    )
+    fig.update_yaxes(rangemode='tozero')
+    fig.write_image(plot_filepath.with_name(plot_filepath.name.replace('runtime', 'runtime_per_minute')))  # noqa
```

Comment:
For some reason, the first figure is correctly called `runtime__ltdb__2021_08_21__19_49_33.csv`, but the second figure is missing the `db_slug` part because it is called `runtime_per_minute__2021_08_20__10_44_04.svg`.

Now that the first plot is computed much faster, I am now tending towards dropping the runtime per minute figure again. Sorry about that, but it makes this file simpler again.

Reply:
I'm not sure what you mean by "the first plot being computed faster"? But agreed, it's gone.

Reply:
Never mind, I don't know what I meant either 😄 – the important thing is that the second figure is now gone again!